ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/view/subviews/ToolTestDataEditor.java

```diff
@@ -0,0 +1,87 @@
+package org.iplantc.de.admin.desktop.client.toolAdmin.view.subviews;
+
+import org.iplantc.de.client.models.tool.ToolAutoBeanFactory;
+import org.iplantc.de.client.models.tool.ToolTestData;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.editor.client.Editor;
+import com.google.gwt.editor.client.SimpleBeanEditorDriver;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.uibinder.client.UiHandler;
+import com.google.gwt.user.client.ui.Widget;
+
+import com.sencha.gxt.widget.core.client.Composite;
+import com.sencha.gxt.widget.core.client.button.TextButton;
+import com.sencha.gxt.widget.core.client.event.SelectEvent;
+
+
+public class ToolTestDataEditor extends Composite implements Editor<ToolTestData> {
+
+    interface ToolTestDataEditorBinder extends UiBinder<Widget, ToolTestDataEditor> {
+    }
+
+    interface EditorDriver extends SimpleBeanEditorDriver<ToolTestData, ToolTestDataEditor> {
+    }
+
+    private static ToolTestDataEditorBinder uiBinder = GWT.create(ToolTestDataEditorBinder.class);
+
+    private final EditorDriver editorDriver = GWT.create(EditorDriver.class);
+
+    //    @Ignore
+//    @UiField TextButton addParameterButton;
+//    @UiField ToolTestDataParamsListEditor paramsEditor;
```

Comment:
From what I understand, the implementation's params aren't stored anymore in the database, only input_files and output_files.  I thought perhaps leaving this here would be helpful if ever params were added back.  I can see how removing it would be cleaner though.